tests/validation/test_cat_vrs_schema.py

```diff
@@ -48,22 +49,22 @@ def _update_cat_vrs_schema_mapping(
 
 
 CAT_VRS_SCHEMA_MAPPING = {schema: CatVrsSchemaMapping() for schema in CatVrsSchema}
-SUBMODULES_DIR = Path(__file__).parents[2] / "submodules" / "cat_vrs" / "schema"
+SUBMODULES_DIR = (
+    Path(__file__).parents[2] / "submodules" / "cat_vrs" / "schema" / "cat-vrs"
+)
 
+with (SUBMODULES_DIR / "cat-vrs-source.yaml").open() as f:
+    cat_vrs_data = yaml.safe_load(f)
+    cat_vrs_defs_keys = cat_vrs_data["$defs"].keys()
 
-# Get core + profiles classes
-for child in SUBMODULES_DIR.iterdir():
-    child_str = str(child)
-    if child_str.endswith(CatVrsSchema.CAT_VRS):
-        mapping_key = CatVrsSchema.CAT_VRS
-    elif child_str.endswith(CatVrsSchema.RECIPES):
-        mapping_key = CatVrsSchema.RECIPES
-    else:
+for f in (SUBMODULES_DIR / "json").glob("*"):
+    if f.name.startswith("example"):
```

Comment:
Does this mean that you are running validation against all json examples/ now? 👀 🥳

Reply:
Ope. No. Great idea tho. Let me do this before merging.

Reply:
That makes sense. I think that we're holding off on merging it for the time being because Daniel wanted Alex to look at it, specifically the "maxItems" property on constraints used to satisfy different Recipes. So, moving it to a new Issue sounds great :)